setup-extension/action.yml

```diff
@@ -111,7 +111,24 @@ runs:
       with:
         path: custom/plugins/${{ inputs.extensionName }}
 
+    - name: Determine whether we are dealing with a plugin or an app
+      shell: bash
+      env:
+        IS_PLUGIN: ${{ hashFiles(format('custom/plugins/{0}/composer.json', inputs.extensionName)) != '' }}
+        IS_APP: ${{ hashFiles(format('custom/plugins/{0}/manifest.xml', inputs.extensionName)) != '' }}
```

Comment:
I think none of our apps the manifest.xml is placed in root btw

Reply:
Thanks, good to know! I'll try implementing a more bulletproof way of detecting those

Reply:
Hi, @shyim it seems like we explicitly don't support anything but a `manifest.xml` in the apps root directory: https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/App/Lifecycle/AppLoader.php#L84-L84

Am I missing something?

Reply:
`$this->appDir` would be `custom/apps` here, I suppose.

Reply:
ahh nvm then